pkgs/native_toolchain_c/lib/src/cbuilder/run_cbuilder.dart

```diff
@@ -161,8 +157,31 @@ class RunCBuilder {
             '-fno-PIC',
             '-fno-PIE',
           ],
+        if (std != null) '-std=$std',
+        if (cpp) ...[
+          '-x',
+          'c++',
+          '-l',
+          cppLinkStdLib ?? defaultCppLinkStdLib[target.os]!
+        ],
+        ...flags,
         for (final MapEntry(key: name, :value) in defines.entries)
           if (value == null) '-D$name' else '-D$name=$value',
+        for (final include in includes) '-I${include.toFilePath()}',
+        ...sources.map((e) => e.toFilePath()),
+        if (executable != null) ...[
+          '-o',
+          outDir.resolveUri(executable!).toFilePath(),
+        ],
+        if (dynamicLibrary != null) ...[
+          '--shared',
+          '-o',
+          outDir.resolveUri(dynamicLibrary!).toFilePath(),
+        ] else if (staticLibrary != null) ...[
+          '-c',
+          '-o',
+          outDir.resolve('out.o').toFilePath(),
```

Comment:
Shouldn't this be `outDir.resolve(staticLibrary).toFilePath(),`? (All our unit tests pass `out.o` as argument?)

Reply:
Ah, I should read me own code :)